### PR TITLE
Implement Buy Crypto UI button in transaction list

### PR DESCRIPTION
**Pull request by amanalili**
- Only appears on BTC, BCH, ETH.
- Added loader when transactions are being loaded.

### CODE REVIEW

**Comment by kylanhurt** [comment]
@paullinator My concern here is that according to @rjrs2k the TransactionList still has issues with some transactions and if the user has one or two transactions but the app thinks it does not then it will show these buttons and make the user even more confused that their transactions aren't showing up.

**Comment by paullinator** [comment]
@kylanhurt I don't think the user will be anymore confused. They will want to see their missing transactions either way.

File: src/modules/UI/scenes/TransactionList/components/BuyCrypto.ui.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const { wallet } = this.props
+    return wallet.currencyNames[wallet.currencyCode]
+  }
+  render () {
+    return (
+      <TouchableWithoutFeedback onPress={Actions.buysell}>
+        <View style={style.buyCryptoContainer}>
+          <View style={style.buyCryptoBox}>
+            <Image style={style.buyCryptoBoxImage} source={{ uri: this.getLogo() }} resizeMode={'cover'} />
+            <T style={style.buyCryptoBoxText}>Buy { this.getCurrencyName() } with Credit Card</T>
```
Comment (kylanhurt) [inline src/modules/UI/scenes/TransactionList/components/BuyCrypto.ui.js]: This string will need to be internationalized. Our Edge team will have to make this change.

File: src/modules/UI/scenes/TransactionList/components/BuyCrypto.ui.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          </View>
+          <View style={style.buyCryptoNoTransactionBox}>
+            <T style={style.buyCryptoNoTransactionText}>No transaction yet!</T>
```
Comment (kylanhurt) [inline src/modules/UI/scenes/TransactionList/components/BuyCrypto.ui.js]: Same here.

**Comment by kylanhurt** [comment]
@amanalili I am not seeing the "Buy Crypto" stuff showing on the txList for empty Transaction Lists. The `emptyListComponent` isn't even being rendered / triggered.

**Comment by paullinator** [comment]
Kylan. Are you checking a BTC BCH Zoe ETH Wallet? It only shows in those

**Comment by kylanhurt** [comment]
@paullinator @swansontec This PR should be ready for merging

**Pull request merged**